pkg/scheduler/schedulerdiscovery/ring_test.go

```diff
@@ -50,7 +51,7 @@ func TestRingConfig_CustomConfigToBasicLifecyclerConfig(t *testing.T) {
 	// ring config
 	expected := ring.BasicLifecyclerConfig{
 		ID:                              "test",
-		Addr:                            "1.2.3.4:10",
+		Addr:                            net.JoinHostPort(cfg.InstanceAddr, strconv.Itoa(cfg.InstancePort)),
```

Comment:
Is this change required?

Reply:
I reverted this change but added a separate test, that independently tests just IPv4 and IPv6.